### PR TITLE
use dev endpoint for non-release, production for release builds

### PR DESCRIPTION
**Pull request by kendase3**

### CODE REVIEW

File: src/meepo.cc [hunk 1]
```diff
@@ -31,9 +32,14 @@ const char path_auth[] = "/auth";
 const char path_sites[] = "/sites";
 const char path_downloads[] = "/downloads";
 
-// TODO(nicholasbishop): this is hardcoded to the staging URL for now
 QUrl createUrl(const QString& path) {
+#ifdef RELEASE
```
Comment (nicholasbishop) [inline src/meepo.cc]: As with the `CHROMEOVER` build variable, we could add an `isRelease()` function to `util.h`.

**Comment by nicholasbishop** [comment]
I was thinkin, since our usage of the meepo api is a pretty boring read-only operation, is using the grv meepo ever really necessary? Maybe we should just hard code the production URL.

**Comment by kendase3** [comment]
I have an account on grv, but not the real one.  I like to use it for testing so I can have multiple sites associated with me, one site, etc.

**Comment by kendase3** [comment]
I'd rather not mess around as an admin on the real meepo

**Comment by kendase3** [comment]
I don't think this is adding much code complexity for that ability

**Comment by nicholasbishop** [comment]
Sure, that sounds reasonable. Just wanted to be sure we needed it.

**Pull request merged**